public/static/locales/en/common.json

```diff
@@ -3,6 +3,14 @@
     "terms": {
       "link": "/en/terms-avis",
       "desc": "Terms and conditions"
+    },
+    "sla": {
+      "link": "/en/sla",
```

Comment:
Adding logic like links in a translation may confuse people. Though we do this in other files like error.json. So I guess it's consistent but just noting for the future :)